src/flowgger/utils/rotating_file.rs

```diff
@@ -143,6 +230,97 @@ impl RotatingFile {
 
         Ok(())
     }
+
+    /// Execute a log file rotation for time triggers
+    /// Create a new file with a timestamped filename. The filename therefore indicates the creation time
+    /// The previous file(s) being also timestamped, they are close
+    ///
+    /// # Returns
+    /// - 'Ok':   when the rotation has been done
+    /// - 'Err':  when the new file could not be open
+    ///
+    fn rotate_time(&mut self) -> io::Result<()> {
+        let _ = writeln!(
```

Comment:
Do you really need this `let _` ?

Reply:
yes, there's warning otherwise because i don't use the writeln status